### ISSUE TITLE
SpatialDropout2D

### ISSUE THREAD

**Issue by tonyreina** [tensorflow/tensorflow#32243]
**System information**
- TensorFlow version (you are using): 1.11
- Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**

SpatialDropout2D is often used in CNNs (https://www.tensorflow.org/api_docs/python/tf/keras/layers/SpatialDropout2D).  At inference time, the layer should be treated just like a regular dropout layer where it just scales the coefficients by the dropout term. However, when I take a Keras model with SpatialDropout2D and saved it as a TF protobuf, TensorFlow converts SpatialDropout2D into a set of primitive operators to do the random dropout instead-- even for a TensorFlow serving graph where the model has been frozen and optimized for inference.  

**Will this change the current api? How?**

I'd like to see the SpatialDropout layers behave the same as the regular Dropout layers during inference.  So the TensorFlow Serving protobuf should not include SpatialDropout2D (or any set of random/slice ops to reconstruct it), but instead should just scale the convolutional filters by the dropout rate as explained in the original paper (https://arxiv.org/abs/1411.4280).

**Who will benefit with this feature?**

If the SpatialDropout layers behave the same as the regular Dropout layers, it will reduce the number of operations during inference and speed up inference on these graphs.

**Any Other info.**

**Comment by dynamicwebpaige** [comment]
This sounds like it might be an implementation bug, rather than a feature request.

@karmel @fchollet, could you take a look?

**Comment by sushreebarsa** [comment]
@tonyreina We see that you are using old version of tensorflow which is officially considered as end of life, We recommend that you upgrade to 2.4 or later version and let us know if the issue still persists in newer versions.Please open a new issue in case you face any errors, we will get you the right help .Hence moving this to closed status.Thanks!
